style.css

```diff
@@ -145,20 +139,149 @@ header {
   align-items: center;
 }
 
-body {
-  background:
-
-    url(img/Ellipse3.png) no-repeat 320px 145px,
-    url(img/Rectangle4.png) no-repeat 290px 115px,
-    url(img/Ellipse8.png) no-repeat 200px 110px,
-    url(img/Rectangle7.png) no-repeat 110px 165px,
-    url(img/Ellipse5.png) no-repeat 110px 175px,
-    url(img/Rectangle6.png) no-repeat 5px 115px,
-    url(img/Ellipse11.png) no-repeat 5px 140px,
-    url(img/Ellipse12.png) no-repeat 300px 535px,
-    url(img/Ellipse13.png) no-repeat 290px 546px,
-    url(img/Ellipse4.png) no-repeat 250px 640px,
-    url(img/Rectangle5.png) no-repeat 200px 638px;
+@media screen and (min-width: 600px) {
+  .right-section {
+    display: none;
+  }
+
+  .time-section {
+    display: none;
```

Comment:
- [ ] This being the desktop version, the min-width (break point) should be in the range of 992px - 1100px.